Firestore/core/src/firebase/firestore/model/snapshot_version.h

```diff
@@ -33,10 +33,11 @@ namespace model {
  */
 class SnapshotVersion : public util::Comparable<SnapshotVersion> {
  public:
-#if __OBJC__
-  SnapshotVersion() {
-  }
-#endif  // __OBJC__
+  /**
+   * Creates a default SnapshotVersion equivalent to SnapshotVersion::None().
+   * Prefer SnapshotVersion::None() for readability.
```

Comment:
(Same comment as before, but again; I suspect this is already correct as is. Likely no action required.)

Reply:
Same response :-). Preventing default construction makes these types very hard to use in practice.